src/services/sendEmail.ts

```diff
@@ -0,0 +1,108 @@
+import { SendEmailCommand, SESClient } from "@aws-sdk/client-ses";
+import { render } from "@react-email/render";
+
+import prisma from "@/lib/prisma";
+import { User } from "@/types";
+
+import userService from "./user";
+import { RepairRequestEmail } from "../email/emails/repair-request-email";
+import { formatDate } from "../utils";
+
+const sesClient = new SESClient({
+  region: process.env.AWS_REGION,
+  endpoint:
+    process.env.AWS_SES_SWITCH === "ON" ? undefined : "http://localhost:8005"
+});
+
+const createEmailContent = (customerUser: User, requestId: string) => {
+  return repairRequestEmail(customerUser, requestId);
+  // add more email templates here
+};
+
+const repairRequestEmail = async (customerUser: User, requestId: string) => {
+  const repairRequest = await prisma.repairRequest.findUnique({
+    where: { id: requestId }
+  });
+
+  let eventDate = "";
+  if (repairRequest?.eventId) {
+    const event = await prisma.event.findUnique({
+      where: { id: repairRequest?.eventId }
+    });
+    eventDate = formatDate(event?.endDate.toDateString() || "");
+  }
+
+  const customerName = `${customerUser?.firstName || ""} ${
+    customerUser?.lastName || ""
+  }`;
+
+  return render(
+    RepairRequestEmail({
+      customerName: customerName,
+      requestId: requestId,
+      date: formatDate(repairRequest?.requestDate.toDateString() || ""),
+      itemName: repairRequest?.itemType || "",
+      issueDescription: repairRequest?.description || "",
+      estimatedDate: eventDate,
+      emailSignature: "" // TODO: Add email signature
+    })
+  );
+};
+
+const createSendEmailCommand = (
+  customerEmail: string,
+  subject: string,
+  content: string
+) => {
+  return new SendEmailCommand({
+    Source: "repairlabtest@gmail.com",
```

Comment:
Add the email as an environment variable

Reply:
Added the 1.mock endpoint; 2. souce email address; 3. mock switch to .env.local.example file as environment variavles